packages/shylu/shylu_dd/frosch/src/Tools/FROSch_ExtractSubmatrices_def.hpp

```diff
@@ -99,7 +99,7 @@ namespace FROSch {
         //cout << *subdomainMatrix << endl;
         RCP<const Comm<LO> > SerialComm = rcp(new MpiComm<LO>(MPI_COMM_SELF));
         RCP<Map<LO,GO,NO> > localSubdomainMap = MapFactory<LO,GO,NO>::Build(map->lib(),map->getNodeNumElements(),0,SerialComm);
-        RCP<Matrix<SC,LO,GO,NO> > localSubdomainMatrix = MatrixFactory<SC,LO,GO,NO>::Build(localSubdomainMap,globalMatrix->getNodeMaxNumRowEntries());
+        RCP<Matrix<SC,LO,GO,NO> > localSubdomainMatrix = MatrixFactory<SC,LO,GO,NO>::Build(localSubdomainMap,globalMatrix->getGlobalMaxNumRowEntries());
```

Comment:
Did this cause a problem at some point?

Reply:
It caused a memory error; now I could not reproduce the error and it works fine with `globalMatrix->getNodeMaxNumRowEntries())`

Reply:
Using multiple processes on `CoarseSolveComm_` I  could reproduce the error: `Tpetra::CrsGraph<int, long long, Kokkos::Compat::KokkosDeviceWrapperNode<Kokkos::Serial, Kokkos::HostSpace> >::insertGlobalIndicesImpl: Proc 0: Not enough capacity to insert 24 indices into local row 9, which currently has 0 entries and total allocation size 16`